src/components/Card.ts

```diff
@@ -47,11 +47,13 @@ export const InnerCardContainer = styled.div<IInnerCardContainer>`
         padding: 3rem;
     `}
 
-    ${props => props.borderBottom && css`
-        border-bottom: 3px solid #000000;
+    ${(props) =>
+        props.borderBottom &&
+        css`
+            border-bottom: 3px solid #000000;
```

Comment:
Not absolutely necessary, but I think for the sake of consistency It'd be very useful to adopt `4px` increments instead of not sticking to any grid for the sake of "pixel perfect design"—things actually start looking better if the spacing is consistent.

Let's add a `GU` variable to theme and use it wherever we have to use pixels. :)

Reply:
Fixed in 71b12631f095a0169dd86908bf04935aeec207ef